crates/typed-store/src/rocks/util.rs

```diff
@@ -40,6 +40,10 @@ pub fn empty_compaction_filter(_level: u32, _key: &[u8], value: &[u8]) -> Compac
     }
 }
 
+pub fn is_ref_count_value(value: &[u8]) -> bool {
+    value.is_empty() || value.len() == 8
```

Comment:
does this depend on full entries always being longer than 8 bytes?

Reply:
full entry always has reference count field itself which is 8 bytes